src/dotnet/CarbonAware.Plugins.NewBasicJsonPlugin/CarbonAware.Plugins.NewBasicJsonPlugin.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <ProjectReference Include="..\CarbonAware\CarbonAware.csproj" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
The rest of this project is all dotnet 6 yes?

Reply:
Yes